src/models/pieChart.js

```diff
@@ -169,7 +169,10 @@ nv.models.pieChart = function() {
             var pieWrap = g.select('.nv-pieWrap').datum([data]);
             d3.transition(pieWrap).call(pie);
 
+            //============================================================
             // Event Handling/Dispatching (in chart's scope)
+            //------------------------------------------------------------
```

Comment:
About this, I am just copying the style from the other source files. For example:
https://github.com/nvd3-community/nvd3/blob/development/src/models/lineChart.js#L236